### ISSUE TITLE
issue with AISLER PCB ordering

### ISSUE THREAD

**Issue by nlooije** [drspangle/infidel-sensor#36]
When ordering PCB on AISLER (accessed through the kitspace website) the PCB apparantly isn't loaded properly:

- PCB, Stencil tabs don't load, are in infinite loop getting production date
- Parts aren't loaded from the BOM

This issue is solved by downloading the Github repo and manually uploading the EAGLE file on AISLER

**Comment by drspangle** [comment]
Paging @darigovresearch ...

**Comment by darigovresearch** [comment]
@kasbah is this a known issue for AISLER?

**Comment by kasbah** [comment]
Thanks for reporting and ating me. Seems to be the same if you download the Gerber zip and upload manually. The Gerbers look OK except the outline doesn't show with Tracespace ([link](https://tracespace.io/view/?boardUrl=https://kitspace.org/boards/github.com/drspangle/infidel-sensor/infidel-sensor-60b6436-gerbers.zip)). The outline does show up when using Gerbv. There also seems some copper outside the outline, but I feel like this should still be able to work. 

Maybe @PatrickFranken has some insight why the Aisler preview can't cope with the Gerbers ([download here](https://kitspace.org/boards/github.com/drspangle/infidel-sensor/infidel-sensor-60b6436-gerbers.zip)) but can cope with the Eagle file.

**Comment by kasbah** [comment]
Just a note: different gerber viewers render the outline differently. 

E.g. OSHPark
![image](https://user-images.githubusercontent.com/206854/116913606-9ce46e80-ac41-11eb-8d05-63943a5c8caa.png)

vs PCBWay (which looks very similar to how Kitspace renders it, they may be using Tracespace as well)
![image](https://user-images.githubusercontent.com/206854/116913675-acfc4e00-ac41-11eb-902a-fa074193a3b2.png)

**Comment by PatrickFranken** [comment]
It looks like this is not a general issue but rather limited to this project. To validate this, I double checked the Gerber files for this project. Please note that our Gerber preview is a bit different to other Gerber viewers as it displays the actual processed manufacturing files compared to others which just render the Gerber files. This also makes it a bit more "critical".
In this case the issue is caused by an indistinguishable, double drawn board outline, see screenshot attached. This makes it impossible to resolve *the correct* milling path for this PCB. As soon as this issue is solved the import should work as expected.
![Screenshot from 2021-05-04 09-36-45](https://user-images.githubusercontent.com/8692465/116973808-5cc2d180-acbd-11eb-9429-1e8bc52564eb.png)

**Comment by darigovresearch** [comment]
Good catch @PatrickFranken, is there anyone from the maintainers of the repository or the community able to resolve this issue in the eagle & KiCad files?

**Comment by CCMCAGP** [comment]
Hi,
Related to this issue, I ordered via Aisler by uploading the EAGLE file manually, which gave this rendering:
![image](https://user-images.githubusercontent.com/50485771/121850706-74926a00-cced-11eb-9385-606f485cd5c8.png)
and I sent the design, but the tests post-manufacturing indicate a big number of opens: 
![image](https://user-images.githubusercontent.com/50485771/121851323-41040f80-ccee-11eb-833e-e0d7a4cbf2e8.png)
 Can anyone indicate to the newbie that I am whether this is expected or not?  
 Thank you!

**Comment by PatrickFranken** [comment]
We are currently seeing quite a lot of false-positive open connections after electrical test of the PCBs. I assume this is also the case here. As the PCB itself is rather simple I expect the manufacturing to be done properly.
